src/client/apiClient.ts

```diff
@@ -46,14 +48,25 @@ const seriesParams = (runId: string, scenario: string) => ({
   metric: "usrActive"
 });
 
-const getJson = async <T>(
+const checkCloudCompatibility = async (client: http.HttpClient, conf: ApiClientConfig): Promise<void> => {
+  const clientName = "gatling-enterprise-github-action";
+  const version = "0.0.1";
+  const response = await client.get(buildUrl(conf, "/compatibility", { clientName, version }));
+  if (response.message.statusCode === HttpCodes.BadRequest) {
+    throw new Error(
+      `Plugin ${clientName} version ${version} is no longer supported; please upgrade to the latest version`
```

Comment:
> please upgrade to the latest version

The latest version is not in the body?

Reply:
Not at the moment, see `io.gatling.enterprise.hexagon.core.domain.CompatibilityService#checkCompatibility` which doesn't do anything yet and only returns an empty response.

Reply:
Again: what a shame :disappointed: